src/app/projects/new-project/new-project.component.html

```diff
@@ -0,0 +1,29 @@
+<form [formGroup]="projectForm" class="new-project-form">
+  <div *ngIf="errorMessage" class="new-project-error">{{errorMessage}}</div>
+  <div fxLayout="column">
+    <div fxFlex>
+      <mat-form-field class="new-project-form-field">
+        <input matInput id="new-project-name" placeholder="Name" formControlName="name" [required]="true">
+      </mat-form-field>
+    </div>
+    <div fxFlex>
+      <mat-form-field class="new-project-form-field">
+        <input matInput placeholder="Customer" formControlName="customer">
+      </mat-form-field>
+    </div>
+    <div fxFlex>
+      <mat-form-field class="new-project-form-field">
+        <input matInput placeholder="Industry sector" formControlName="industrySector">
+      </mat-form-field>
+    </div>
+    <div fxFlex>
+      <mat-form-field class="new-project-form-field">
+        <input matInput placeholder="Description" formControlName="description">
+      </mat-form-field>
+    </div>
+  </div>
+  <div class="new-project-actions">
```

Comment:
There is no `new-project-actions` class in `new-project.component.scss`. Do we need the class here?

Reply:
done